modules/framework/zio/test/src/weaver/ziocompat/MutableZIOSuiteTest.scala

```diff
@@ -75,6 +75,33 @@ object ZIOSuiteTest extends ZIOSuite[KVStore with DogFoodz] {
     }
   }
 
+  test("logs contain only the logs from each test") {
+    FiberRefLogTest.spec(List.empty)
```

Comment:
Is this the kind of test you were after @Baccata ?

Reply:
Yes, but I'd also like some tests showcasing the benefit of using a FiberRef based solution (you could re-use the example you have in the description of this PR)